hmda-data-publisher/src/test/scala/hmda/publisher/helper/QuarterTimeBarrierTest.scala

```diff
@@ -9,6 +9,8 @@ class QuarterTimeBarrierTest extends FreeSpec {
 
   val  quarters2020 =List(Period.y2020Q1,Period.y2020Q2,Period.y2020Q3)
   val  quarters2021 =List(Period.y2021Q1,Period.y2021Q2,Period.y2021Q3)
+  val  quarters2022 =List(Period.y2022Q1,Period.y2022Q2,Period.y2022Q3)
+  val  quarters2023 =List(Period.y2023Q1,Period.y2023Q2,Period.y2023Q3)
```

Comment:
These vals are never used, can you remove them?

Reply:
When you removed them, did your unit tests pass? Mine fails:

`/Users/jaredbarrow/Desktop/projects/hmda_plat/hmda-data-publisher/src/main/scala/hmda/publisher/helper/QuarterTimeBarrier.scala:3:8
object YearPeriod is not a member of package hmda.publisher.query.component
import hmda.publisher.query.component.YearPeriod`

I believe that the tests are somehow grabbing the quarter from these lists, I'm not entirely sure though

Reply:
I tried removing them and the tests passed in sbt for me. These lists of periods are never used in our code as far as I can tell: https://github.com/search?q=repo%3Acfpb%2Fhmda-platform%20quarters2020&type=code

Reply:
Oh sorry, I've been using intellij to run single unit tests. On sbt it passes, removed the quarters.